1.traverseLinkedList.js

```diff
@@ -19,8 +19,17 @@
 // [556, 41, 33, 150, 11]
 
 
-function traverse(list) {
 
+const traverse = (list) => {
+    if(!list) return []
```

Comment:
This extra check isn't needed. If there's no list you'll never go into the while loop.